README.md

```diff
@@ -24,6 +26,25 @@ sudo apt install facerec
 source ~/.bashrc
 ```
 
+### Using .deb package
+
+#### 1. Download the latest release of Facerec from the releases section of the repository
```

Comment:
` wget https://github.com/rushabh-v/linux_face_unlock/releases/download/v1.1/facerec_1.1_all.deb` to download it.